packages/odyssey-storybook/src/components/odyssey-labs/SideNav/SideNav.stories.tsx

```diff
@@ -0,0 +1,302 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { SideNav, SideNavProps } from "@okta/odyssey-react-mui/labs";
+import { Meta, StoryObj } from "@storybook/react";
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+import {
+  AppsIcon,
+  ClockIcon,
+  SettingsIcon,
+  CopyIcon,
+  HomeIcon,
+  Fido2Icon,
+  LockIcon,
+  BugIcon,
+  CloseIcon,
+  CalendarIcon,
+  CallIcon,
+  CheckIcon,
+  AddCircleIcon,
+  DownloadIcon,
+  ChatIcon,
+  RefreshIcon,
+  UserIcon,
+  DirectoryIcon,
+  GlobeIcon,
+  IdpIcon,
+  InformationCircleIcon,
+  InformationCircleFilledIcon,
+  ServerIcon,
+  ExpandLeftIcon,
+  FolderIcon,
+} from "@okta/odyssey-react-mui/icons";
+
+const storybookMeta: Meta<SideNavProps> = {
+  title: "Labs Components/SideNav",
+  component: SideNav,
+  argTypes: {
+    navHeaderText: {
+      control: "text",
+      description: "Header text for the side nav",
+      table: {
+        type: {
+          summary: "string",
+        },
+      },
+    },
+    isCollapsible: {
+      control: "boolean",
+      description: "Controls whether the side nav is collapsible",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+      },
+    },
+    onCollapse: {
+      description: "Callback to be triggered when the side nav is collapsed",
+    },
+    sideNavItems: {
+      description: "",
+      table: {
+        type: {
+          summary: "Array<SideNaItem>",
+        },
+      },
+    },
+    footerItems: {
+      description: "",
+      table: {
+        type: {
+          summary: "Array<SideNavFooterItem>",
+        },
+      },
+    },
+  },
+  args: {
+    navHeaderText: "Admin",
+    isCollapsible: true,
+    onCollapse: () => {},
```

Comment:
We shouldn't have to define `onCollapse` as a prop. Storybook will pass that in for you and log it to the Actions tab.